frontik/handler.py

```diff
@@ -305,6 +305,16 @@ def set_status(self, status_code, reason=None):
         super().set_status(status_code, reason=reason)
 
     def redirect(self, url, *args, **kwargs):
```

Comment:
```suggestion
    def redirect(self, url, *args, allow_protocol_relative=False, **kwargs):
```
Давай вот так. Я ещё тебя с True обманул, прости. Мы хотим, чтобы по умолчанию было нельзя

Reply:
это сразу сделал, там not в if